pyproject.toml

```diff
@@ -10,6 +10,8 @@ flask = "^1.1.2"
 python-dotenv = "^0.14.0"
 requests = "^2.13.0"
 gunicorn = "^20.1.0"
+pymongo = {extras = ["srv"], version = "^3.12.0"}
+mongomock = "^3.23.0"
```

Comment:
I don't think you use mongomock. If you want to use it for the integration test then go ahead, but otherwise you can remove it